python_modules/dagster/dagster/_core/definitions/resource_annotation.py

```diff
@@ -30,4 +30,4 @@ def _is_resource_annotated(param: Parameter) -> bool:
 
 
 T = TypeVar("T")
-Resource = Annotated[T, "resource_output"]
```

Comment:
can you change "resource_output" to "resource_param" while you are at it make it a symbol since it is reference elsewhere